kivy/core/window/_window_sdl2.pyx

```diff
@@ -135,7 +135,7 @@ cdef class _WindowSDL2Storage:
         SDL_GL_SetAttribute(SDL_GL_RED_SIZE, 8)
         SDL_GL_SetAttribute(SDL_GL_BLUE_SIZE, 8)
         SDL_GL_SetAttribute(SDL_GL_GREEN_SIZE, 8)
-        SDL_GL_SetAttribute(SDL_GL_ALPHA_SIZE, 8)
+        SDL_GL_SetAttribute(SDL_GL_ALPHA_SIZE, int(environ.get('KIVY_SDL_GL_ALPHA_SIZE', '8')))
```

Comment:
I don't like the variable name `KIVY_SDL_GL_ALPHA_SIZE` because when someone wants to use console mode in the pi, they should be setting a variable e.g. `KIVY_WINDOW_PI_CONSOLE=1` rather than `KIVY_SDL_GL_ALPHA_SIZE`, which is something internal.

However, can't we always set this to zero on the Pi? Why is this user configurable? Could we not set it to zero when the platform is pi as detected in `setup.py`?

Reply:
@matham yes I would prefer to always set that to 0 on the Pi 4 and forward. I just didn't want to break anything, that's why I added that variable.

Reply:
See: https://github.com/kivy/kivy/pull/6769/commits/a0ff5212d5e2a60deb13cc9e7165df141d2df99b. Let me know if you want me to do it differently or use another name for the option.

Reply:
Yeah, this seems ok to me.

Reply:
Okay super. Let me clean it up and test the wheels on RPi 3 and RPi 4, so I can make sure the change does not break anything. I hope to do that tomorrow.